pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -61,10 +66,37 @@ func (az *Cloud) GetLoadBalancer(clusterName string, service *v1.Service) (statu
 	}, true, nil
 }
 
+func (az *Cloud) getPublicIPName(clusterName string, service *v1.Service) (string, error) {
+	loadBalancerIP := service.Spec.LoadBalancerIP
```

Comment:
Would this change mean the user always has to bring his own IP? If so, it would probably be a breaking change.